android/build.gradle

```diff
@@ -431,7 +431,9 @@ task cleanNdkLibs(type: Exec) {
 task packageNdkLibs(dependsOn: buildReanimated, type: Copy) {
     from("$buildDir/reanimated-ndk/all")
     include("**/libreanimated.so")
-    include("**/libturbomodulejsijni.so")
+    if(System.getenv('REACT_NATIVE_TARGET_VERSION').toInteger() < 64) {
```

Comment:
You can get RN version from `reactProperties ` as we already do with Folly version. `VERSION_NAME` property is what you need here. I think the environment variable is problematic because we have it only in script.